<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>801</height>
    </rect>
   </property>
   <widget class="QPushButton" name="gameButtonStart">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelMyScore">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>My score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabelPlayer7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wylosowana litera:</string>
    </property>
   </widget>
   <widget class="QLabel" name="letterLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="countryTextEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="cityTextEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Panstwo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Miasto</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameTextEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Imie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameButtonSend">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyslij odpowiedzi</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheckMyScore">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck8">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameTimerLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
